docs/Formulas.md

```diff
@@ -0,0 +1,69 @@
+# How pe_tune calculates values
+
+## Memory
+
+### Minimums and Maximums
```

Comment:
Some of these are subject to change depending upon the characteristics of the node or infrastructure.